alica_engine/src/engine/RunningPlan.cpp

```diff
@@ -53,6 +51,12 @@ RunningPlan::RunningPlan(AlicaEngine* ae)
         , _basicBehaviour(nullptr)
         , _parent(nullptr)
 {
+    std::call_once(once, []() {
```

Comment:
Hm, can we move this out of the constructor, there is no reason to test this every time in constructor, nor for all the threadsafe machinery that this entails.

Reply:
It's thread-safe, there is only one thread that create (and deletes) RunningPlans, but imo even a bool flag is unnecessary overhead.

Reply:
Rolled back.